api/controllers/AuthController.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * AuthController
+ */
+
+var jwt = require('jwt-simple'),
+    moment = require('moment'),
+    request = require('request'),
+    qs = require('querystring');
+
+
+module.exports = {
+
+  me: function(req, res, next) {
```

Comment:
Not really, this is the endpoint that return the current user. `/api/v1/me` will return your user. However, we can totally change the endpoint name :P